goverge/coverage.py

```diff
@@ -32,11 +32,11 @@ def check_failed(return_code):
 
 def generate_coverage(
         packages, project_package, project_root, godep, short, xml, xml_dir,
-        race, tag, max_threads, go_flags):
+        race, tag, max_threads, go_flags, ignore):
```

Comment:
Since all references to ignore are not being used in this file can you remove it please?